sio-digital-team/active-safe
# Acf setting update

https://trello.com/c/b0M0TVyJ

**Changes:**
- [x] Fixed ACF Plugin asset reference for hotspots
- [x] Updated Risk Factor fluid container
- [x] Added Grid feature

**How to Test:**
- Create a new page with hotspots and see if you can add the location of the hotspot
- Check the page to see if risk factors don't have too much empty space below it
